src/main/java/apod/APODFrame.java

```diff
@@ -99,11 +104,20 @@ public void setPhoto(String photoUrl){
         }
     }
 
+//    public void setVideo(String videoURL) {
+//
+//    }
+
```

Comment:
remove commented out code.

Reply:
I wanted to make an attempt at embedding a video in the gui since on occasion, the api returns a video instead of an image for that day